homeassistant/components/islamic_prayer_times/__init__.py

```diff
@@ -41,6 +41,18 @@ def update_unique_id(
     return True
 
 
+async def async_migrate_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> bool:
```

Comment:
In the new config flow you are also setting an unique_id, I think it would be wise to apply this to the migration as well.

Reply:
done